lib/src/model/documentation_comment.dart

```diff
@@ -97,14 +97,11 @@ mixin DocumentationComment on Documentable, Warnable, Locatable, SourceCode {
   /// result.
   String _processCommentWithoutTools(String documentationComment) {
     var docs = stripComments(documentationComment);
-    if (!docs.contains('{@')) {
-      _analyzeCodeBlocks(docs);
-      return docs;
+    if (docs.contains('{@')) {
```

Comment:
This changes causes us to execute `_stripHtmlAndAddToIndex` for docs which do not contain `{@`. But we know such docs don't contain `{@inject-html`, so it seems there is no reason to do that work?

Reply:
done